config/features.yml

```diff
@@ -21,6 +21,10 @@ features:
     actor_type: user
     description: >
       Allows user to access chapter 33 benefits direct deposit view/update APIs
+  covid_vaccine_registration:
+    actor_type: user
+    description: >
+      Toggles availability of covid vaccine registration form frontend and API.
```

Comment:
FYI that per a Slack thread I added a separate Flipper feature for the frontend app, https://github.com/department-of-veterans-affairs/vets-api/pull/5448